### ISSUE TITLE
New release wanted

### ISSUE THREAD

**Issue by rom1504** [NightTsarina/python-rocksdb#17]
hi,
I'm finding that rocksdb python package is outdated
do you plan to update it ?

what are the changes here?

**Comment by NightTsarina** [comment]
Hi Romain, I am not sure I understand.. Could you explain what is the issue you are reporting?

**Comment by rom1504** [comment]
I think it would be great if there was a pypi package with an up to date version of rocksdb.

It seems your fork is currently the most up to date.
Could you consider publishing it to pypi?

**Comment by NightTsarina** [comment]
Yes, I am planning on preparing a release soon, after some tests are finished. And I will look into publishing to pypi too

**Comment by wbarnha** [comment]
I really like the work done in this repository. It'd be nice if we could set this as 'head' for all the other forks of python-rocksdb or get this more attention. Looking forward to the release!

**Comment by feevos** [comment]
Great work, SO useful!!! Thank you very much for your contribution!!

**Comment by eladn** [comment]
Thanks for maintaining this project! I find RocksDB usable for many python-based applications. I wonder how come it's not very known.
